artemis-commons/pom.xml

```diff
@@ -53,6 +71,7 @@
       <dependency>
           <groupId>com.google.errorprone</groupId>
           <artifactId>error_prone_core</artifactId>
+         <scope>provided</scope>
```

Comment:
Did this make a difference? The dependencyManagement is setting the scope so it should already be at an effective provided. (Mainly just curious why you actually changed it, personally I do prefer each module states it)

Reply:
I was playing between provided and compile..

The issue with compiled is shade will not pick it up.

Shade will generate another PR, but maven will not look at it internally. It's a bit of a mess I'm not being able to figure out.

Reply:
So yes, I see a comment in the netty codebase that shading needs compile scope. Plus it does seem that the shading actually removes the dep, so....no idea why you are seeing issues.